src/testing/helpers.ts

```diff
@@ -0,0 +1,54 @@
+import { DebugElement } from '@angular/core';
+import { Pipe, PipeTransform } from '@angular/core';
+import { MockHalDoc } from 'ngx-prx-styleguide';
+
```

Comment:
Some of the helper stuff is fairly particular to Publish, thinking specifically of stuff like `create` and `cit` where it sets up the testing module with Cms and Feeder services that live in Publish, but maybe we should at least be identifying the helpers that we are duplicating to be added to a common place in the styleguide?

Reply:
Yes, that sounds good. -> https://github.com/PRX/styleguide.prx.org/issues/81